lib/browser.ts

```diff
@@ -672,17 +673,14 @@ export class ProtractorBrowser extends Webdriver {
 
     if (this.ignoreSynchronization) {
       this.driver.get(destination);
-      return this.driver.controlFlow().execute(() => {
-        return this.plugins_.onPageLoad();
-      });
+      return this.driver.controlFlow().execute(() => this.plugins_.onPageLoad()).then(() => {});
```

Comment:
These functions are exposed indirectly. `ProtractorBrowser#get` is exposed. The very line we're discussing now is its return statement. So it's not just type checking, it's public API.